### PR TITLE
docs: add link to laboon project

### PR DESCRIPTION
**Pull request by arisnacg**
Hello, Bubbletea team. Thank you for your great library 🎉

I created a Bubbletea project for managing Docker containers that look like Docker Desktop. I added the link in the Wild section of the README. 

Preview of my Bubbletea project:

laboon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -393,6 +393,7 @@ For some Bubble Tea programs in production, see:
 * [wander](https://github.com/robinovitch61/wander): a HashiCorp Nomad terminal client
 * [WG Commander](https://github.com/AndrianBdn/wg-cmd): a TUI for a simple WireGuard VPN setup 
 * [wishlist](https://github.com/charmbracelet/wishlist): an SSH directory
+* [laboon](https://github.com/arisnacg/laboon): an TUI that similliar Docker Desktop
```
Comment (meowgorithm) [inline README.md:396]:
```suggestion
* [laboon](https://github.com/arisnacg/laboon): a Docker-desktop-style container manager
```

**Review by meowgorithm — CHANGES_REQUESTED** [review]
This looks great! In addition to the copy edit suggestion, do you mind putting this in alphabetical order?

File: README.md (hunk 1 above)
Comment (arisnacg) [reply README.md:396]: Sure!

**Comment by meowgorithm** [comment]
Awesome, thank you!

**Pull request merged**